frontend/src/layout/navigation/TopBar/TopBar.tsx

```diff
@@ -47,26 +43,22 @@ export function TopBar(): JSX.Element {
                         <FriendlyLogo />
                     </Link>
 
-                    {featureFlags[FEATURE_FLAGS.UNIVERSAL_SEARCH] ? (
```

Comment:
Does this constant need to be removed?

Reply:
yes indeed. No wonder I couldn't find it in search: I deleted it but didn't commit -.- -.- -.- -.-